### PR TITLE
Text Adventure

### PR DESCRIPTION
**Pull request by piperchester**
Merging current status of textAdventure. This includes a new view controller, .plist, header and class file.

### CODE REVIEW

File: AccessBraille/AccessBraille/NavigationView.m [hunk 1]
```diff
@@ -0,0 +1,218 @@
+//
+//  NavigationView.m
```
Comment (7imbrook) [inline AccessBraille/AccessBraille/NavigationView.m]: This file should be removed

File: AccessBraille/AccessBraille/TextAdventure.h [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+//
+//  Created by Piper Chester on 6/3/13.
+//  Copyright (c) 2013 RIT. All rights reserved.
+//
+
+#import <UIKit/UIKit.h>
+#import "ABKeyboard.h"
+#import "ABSpeak.h"
+
+@interface TextAdventure : UIViewController <ABKeyboard> { // Follow the ABKeyboard protocol
```
Comment (7imbrook) [inline AccessBraille/AccessBraille/TextAdventure.h]: Comment not necessary

File: AccessBraille/AccessBraille/TextAdventure.m [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+
+#import "TextAdventure.h"
+
+@interface TextAdventure ()
+
+@end
+
+@implementation TextAdventure
+
+#pragma mark -
```
Comment (7imbrook) [inline AccessBraille/AccessBraille/TextAdventure.m]: Extra pragma

File: AccessBraille/AccessBraille/TextAdventure.m [hunk 4]
```diff
@@ -0,0 +1,168 @@
...
+    infoText = [[UITextView alloc]initWithFrame:CGRectMake(50, 150, 900, 400)];
+    [infoText setText:[texts valueForKey:@"initialText"]];
+    [infoText setFont:[UIFont fontWithName:@"ArialMT" size:40]];
+    [infoText setBackgroundColor:[UIColor clearColor]];
+    [infoText setUserInteractionEnabled:NO];
+    [[self view] addSubview:infoText];
+}
+
+- (void)viewDidAppear:(BOOL)animated {
+    [self.view setFrame:CGRectMake(0, 0, 1024, 768)];
```
Comment (7imbrook) [inline AccessBraille/AccessBraille/TextAdventure.m:50]: should use [UIScreen mainScreen].bounds, avoid using hardcoded size values

File: AccessBraille/AccessBraille/TextAdventure.m [hunk 5]
```diff
@@ -0,0 +1,168 @@
...
+
+
+-(void)changeToRoom:(NSString* )room {
+    if ([room isEqual: @"waterfront"]){
+        currentLocation = [texts valueForKey:@"waterfrontDescription"];
+    }
+}
+
+
+#pragma mark -
```
Comment (7imbrook) [inline AccessBraille/AccessBraille/TextAdventure.m]: Extra pragma

File: AccessBraille/AccessBraille/TextAdventure.m [hunk 6]
```diff
@@ -0,0 +1,168 @@
...
+
+
+#pragma mark - 
+#pragma mark - Keyboard Methods
+
+/**
+ * Speak character being typed, as well as appending it to the UITextView.
+ */
+- (void)characterTyped:(NSString *)character withInfo:(NSDictionary *)info {
+    if ([character isEqual: @" "]){
```
Comment (7imbrook) [inline AccessBraille/AccessBraille/TextAdventure.m]: `[info[ABSpaceTyped] boolValue]`
I need to document this

File: AccessBraille/AccessBraille/TextAdventure.m (hunk 4 above)
Comment (piperchester) [inline AccessBraille/AccessBraille/TextAdventure.m:50]: Yep, I need to get on this...

**Pull request merged**